src/nvim/event/loop.h

```diff
@@ -9,8 +9,8 @@
 
 typedef void *WatcherPtr;
 
-#define _NOOP(x)
-KLIST_INIT(WatcherPtr, WatcherPtr, _NOOP)
+#define NOOP(x)
+KLIST_INIT(WatcherPtr, WatcherPtr, NOOP)
```

Comment:
side note, this is the only place we use klist, could we use kvec instead?

Reply:
@bfredl is the kväckmaster, he'd know